### PR TITLE
Display gist editor url on success of creation or update

### PR DESCRIPTION
**Pull request by algomaster99**
Show the editor Gist URL which is `/#/gist/<gist-id>/<filename>/` whenever updating or creation is successful.

### CODE REVIEW

File: src/components/header/share-modal/renderer.tsx [hunk 1]
```diff
@@ -159,10 +159,11 @@ class ShareModal extends React.PureComponent<Props, State> {
             }
           );
         } else {
+          const BASE_URL = window.location.origin;
+          const {fileName, gistId} = data;
           this.setState({
             createError: false,
-            // TODO: create URL to editor. BASE_URL/#/gist/GIST_ID
-            gistEditorURL: data.url,
+            gistEditorURL: `${BASE_URL}/#/gist/${gistId}/${fileName}/`,
```
Comment (domoritz) [inline src/components/header/share-modal/renderer.tsx:166]:
```suggestion
            gistEditorURL: `${BASE_URL}/#/gist/${gistId}/${fileName}`,
```

File: src/components/header/share-modal/renderer.tsx [hunk 2]
```diff
@@ -200,9 +201,10 @@ class ShareModal extends React.PureComponent<Props, State> {
 
       const data = await res.json();
       if (res.status === 205) {
+        const BASE_URL = window.location.origin;
+        const {fileName, gistId} = data;
         this.setState({
-          // TODO: create URL to editor. BASE_URL/#/gist/GIST_ID
-          gistEditorURL: data.url,
+          gistEditorURL: `${BASE_URL}/#/gist/${gistId}/${fileName}/`,
```
Comment (domoritz) [inline src/components/header/share-modal/renderer.tsx:207]:
```suggestion
          gistEditorURL: `${BASE_URL}/#/gist/${gistId}/${fileName}`,
```

**Comment by algomaster99** [comment]
Will fix the changes in https://github.com/vega/editor/pull/697. Anyway we need to merge that PR first :)

**Comment by algomaster99** [comment]
Resolved here b5bb4a85898753b3709700230402521aefb17e6c

File: src/components/header/share-modal/renderer.tsx (hunk 2 above)
Comment (algomaster99) [reply src/components/header/share-modal/renderer.tsx:207]: Resolved here b5bb4a85898753b3709700230402521aefb17e6c

File: src/components/header/share-modal/renderer.tsx (hunk 1 above)
Comment (algomaster99) [reply src/components/header/share-modal/renderer.tsx:166]: Resolved here b5bb4a85898753b3709700230402521aefb17e6c

**Pull request merged**